Core/src/org/sleuthkit/autopsy/casemodule/events/BlackBoardArtifactTagAddedEvent.java

```diff
@@ -45,7 +46,7 @@ public BlackBoardArtifactTagAddedEvent(BlackboardArtifactTag newTag) {
      * @throws TskCoreException
      */
     @Override
-    BlackboardArtifactTag getTagByID() throws IllegalStateException, TskCoreException {
-        return Case.getCurrentCase().getServices().getTagsManager().getBlackboardArtifactTagByTagID(getTagID());
+    BlackboardArtifactTag getTagByID() throws NoCurrentCaseException, TskCoreException {
+        return Case.getOpenCase().getServices().getTagsManager().getBlackboardArtifactTagByTagID(getTagID());
```

Comment:
This is the right thing to do, but please include the change to the client (TagAddedEvent) in this pull request, so that we keep the incremental changes together.